src/internals/shape.point.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2017 NAVER Corp.
+ * billboard.js project is licensed under the MIT license
+ */
+import ChartInternal from "./ChartInternal";
+import {isFunction, extend} from "./util";
+
+const POINT_TYPES = ["circle", "rectangle"];
+
+extend(ChartInternal.prototype, {
+	hasValidPointType() {
+		return POINT_TYPES.includes(this.config.point_type);
```

Comment:
Array.includes isn't supported in IE. billboard.js has IE9+ support.
- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Can be transpiled using one of babel plugin
- https://www.npmjs.com/package/babel-plugin-array-includes

but, to not have another dev dependency, it might be better change in more compatible way.
```js
     return /^(circle|rectangle)$/i.test(this.config.point_type);
     // or using Array.indexOf()
```

Reply:
Perfect thanks, I've updated my PR with this code.